app/views/shared/_transcriber_templates.html.erb

```diff
@@ -191,29 +191,29 @@
                 <div class="labels-container">
                   <!-- "What is the date and time of this observation? <b>You can select the date and time by clicking on the clock icon.</b><br>Should follow the format <nobr><code>04 Aug 1874, 01:48 AM</code></nobr>." -->
                     <div class="scribe_input_label date-label">
-                      <%= content_tag "i", "", 
-                        class: "fa fa-info-circle", 
-                        style: "cursor: help;", 
-                        title: strip_tags(t('transcriber.transcription-box-template.form.fields.labels.datetime')), 
-                        data: { 
-                          content: t('transcriber.transcription-box-template.form.fields.help.datetime').html_safe, 
-                          html: true, toggle: "popover", placement: "auto", trigger: "hover", container: "body"
-                        } 
-                      %> <%= t('transcriber.transcription-box-template.form.fields.labels.datetime') %>
+                      <label for="datetime-help"
+                      style="cursor: help;"
+                      aria-hidden="true" 
+                      title="<%=strip_tags(t('transcriber.transcription-box-template.form.fields.labels.datetime'))%>"
+                      data-toggle="popover" 
+                      data-trigger="hover" 
+                      data-content="<%= t('transcriber.transcription-box-template.form.fields.help.datetime').html_safe %>" 
+                      data-container="body" 
+                      data-html=true
+                      data-placement="auto"> 
+                      <%= t('transcriber.transcription-box-template.form.fields.labels.datetime') %></label>
                     </div>
                     <% field_group.fields.each do |field| %>
                       <div class="scribe_input_label">
-                        <i 
-                          class="fa fa-info-circle" 
+                        <label for="<%= field.id %>"
```

Comment:
In the `for` property, the ID that was mentioned in the docs is the ID of the `select` element in the form that the label corresponds to. I checked last night, and it doesn't seem like the select tag for the fields have an ID set, so you will have to add on there first. When it comes to HTML element IDs, the IDs are supposed to be unique.

You can re-use classes, but not IDs. I mention this because how it's currently written could produce duplicate DOM IDs if the same thing were done for other records. 

Luckily, Rails provides a handy way to handle this! you can use the `dom_id` method to generate a string in the form of of `<model_name>_<id>` from the `ActionView::RecordIdentifier` module. This should be automatically available in all templates. https://www.rubydoc.info/github/rails/rails/ActionView/RecordIdentifier

In order to use it, you just have to pass a model instance to the `dom_id` method. In this case, you would have `<%= dom_id(field) %>`.  Then you would add an ID attribute to the select tag on [line 258](https://github.com/open-data-rescue/climate-data-rescue/blob/41b98a0c338dc405b8856b330e7948f4bd65bbce/app/views/shared/_transcriber_templates.html.erb#L258) using the same value like so:

```ruby
...
id: dom_id(field),
multiple: field.multi_select, 
placeholder: (field.multi_select ? 'select' : 'type'), 
...
```

Reply:
I fixed this one in the newest commit.